vclib-openid/src/commonTest/kotlin/at/asitplus/wallet/lib/oidc/OidcSiopIsoProtocolTest.kt

```diff
@@ -77,16 +76,13 @@ class OidcSiopIsoProtocolTest : FreeSpec({
             relyingPartyUrl = relyingPartyUrl,
         )
         val document = runProcess(
-            verifierSiop,
-            walletUrl,
-            OidcSiopVerifier.RequestOptions(
+            verifierSiop, walletUrl, OidcSiopVerifier.RequestOptions(
```

Comment:
Please keep the line breaks

Reply:
fixed in commit 3fffbac898b7db1ed394255e34e9fe0ae7cdc665